nginx/conf.d/default.conf

```diff
@@ -60,6 +65,10 @@ server {
         proxy_pass      http://bugfinder/;
     }
 
+    location /minigames/finitequiz/ {
+        proxy_pass      http://finitequiz/;
+    }
+
     location /lecturer-interface/ {
         proxy_pass      http://lecturer-interface/;
     }
```

Comment:
```suggestion
    }
    
    location /third-party-license-notice/ {
        proxy_pass      http://third-party-license-notice/;
    }
```

Reply:
oh sorry i forgot it enen though you remembered me so often :(